package.json

```diff
@@ -39,7 +39,8 @@
     "prop-types": "15",
     "react": "16",
     "react-dom": "16",
-    "react-immutable-proptypes": "2"
+    "react-immutable-proptypes": "2",
+    "styled-components": "4.1.3"
```

Comment:
Peer deps should have more looser version definitions, maybe just "4" would be ok.

Reply:
OK, but it seems that styled-components is a dependency in react-floating-select and some other component and changing it here to peer dep and adding it to externals doesn't really help.